src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicIndexerAccessExpression.cs

```diff
@@ -377,5 +377,563 @@ void M(C c, dynamic d)
 
             VerifyOperationTreeAndDiagnosticsForTest<ElementAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void DynamicIndexerAccess_ControlFlowInReceiver()
```

Comment:
>DynamicIndexerAccess_ControlFlowInReceiver [](start = 20, length = 42)

Can we cover null receiver (is it possible) for IDynamicIndexerAccessOperation?

Reply:
Added `DynamicIndexerAccess_NullReceiver`. It generates compiler errors but contains an invalid `IDynamicIndexerAccessOperation`.

---
In reply to: [186838334](https://github.com/dotnet/roslyn/pull/26690#discussion_r186838334) [](ancestors = 186838334)